### ISSUE TITLE
Motion-list change how motion title are read by Screenreader

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#5599]
**Current behavior**
The motion title in  the motion list are embedded in a div and therefore not read as importance elements

<img width="603" height="174" alt="Image" src="https://github.com/user-attachments/assets/ef2e3343-26ef-4889-a082-ffe1af488b12" />

**Wanted behavior**
Without changing the appearance the motion titles should be a h1

**Additional:**
I would recommmend changing the div to a h1 and then mimicking the current style.
Important: The elipsis for long titles should still work
